clients/src/main/java/org/apache/kafka/common/requests/CreateAclsRequest.java

```diff
@@ -51,9 +52,20 @@
                     PERMISSION_TYPE))));
 
     /**
-     * The version number is bumped to indicate that on quota violation brokers send out responses before throttling.
+     * V1 sees a new `RESOURCE_NAME_TYPE` that controls how the resource name is interpreted and version
+     * was bumped to indicate that, on quota violation, brokers send out responses before throttling.
```

Comment:
I think this could be phrased a little bit better.  Something like "Version 1 adds RESOURCE_NAME_TYPE.  Also, when the quota is violated, brokers will respond to a version 1 or later request before throttling."

Reply:
I thought so to, but didn't want to change other peoples wording! Consider it done.